website/content/en/docs/add-ons/storage/efs/guide.md

```diff
@@ -37,6 +37,8 @@ export CLAIM_NAME=<efs-claim>
 
 ## 2.0 Set up EFS
 
+> Important: If you have deployed Kubeflow using any of the Terraform deployment options and have not set `enable_aws_efs_csi_driver = false` then skip this section.
```

Comment:
Does this mean efs will be setup by default using terraform (`enable_aws_efs_csi_driver=true` by default)?  
If so that's awesome!

Reply:
Yeah, `enable_aws_efs_csi_driver=true` will deploy the appropriate helm chart and create the SA to use EFS